web-components/src/[sandbox]/examples/button-group.ts

```diff
@@ -0,0 +1,43 @@
+import "@/components/button-group/ButtonGroup";
```

Comment:
please add a disclaimer comment to each new file

Reply:
Done